lib/Resources/config/container/solr/sort_clause_visitors.yml

```
@@ -78,6 +78,12 @@ services:
             - {name: ezpublish.search.solr.query.content.sort_clause_visitor}
             - {name: ezpublish.search.solr.query.location.sort_clause_visitor}
 
+    ezpublish.search.solr.query.common.sort_clause_visitor.score:
+        class: EzSystems\EzPlatformSolrSearchEngine\Query\Common\SortClauseVisitor\Score
```

Technical debt
```suggestion
    EzSystems\EzPlatformSolrSearchEngine\Query\Common\SortClauseVisitor\Score:
```

or:
```suggestion
    ibexa.platform.search.solr.query.common.sort_clause_visitor.score:
        class: EzSystems\EzPlatformSolrSearchEngine\Query\Common\SortClauseVisitor\Score
```

---

No, isn't technical dept. Nevertheless class name could be used as service id. See 922feaf